portfolio/src/main/java/com/google/sps/servlets/LoginServlet.java

```diff
@@ -37,25 +40,31 @@
   
   @Override 
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    // Set response type.
-    response.setContentType("text/html");
-    
     UserService userService = UserServiceFactory.getUserService();
+    Map loginStatusInfo = new HashMap();
```

Comment:
Nit: prefer to renaming the variable to be `loginStatusInfoMap` so that subsequent flows can easily/clearly know the variable is a map.